server/src/main/proto/dst_server_protocol.proto

```diff
@@ -6,42 +6,61 @@ option java_package="org.dst.server.generated";
 option java_outer_classname="DstServerProtocol";
 option cc_generic_services = true;
 
-
+//string
 message StringPutRequest {
     required string key = 1;
     required string value = 2;
 };
 
 message StringPutResponse {
-    required string result = 1;
+    required string response = 1;
 };
 
 message StringGetRequest {
     required string key = 1;
 }
 
 message StringGetResponse {
-    required string result = 1;
+    required string response = 1;
+    required string value =2;
 };
 
-
-
+//list
 message ListPutRequest {
     required string key = 1;
-    repeated string values = 2;
+    repeated string value = 2;
```

Comment:
Why change this line?

Reply:
Keep all return value names  same will be more readable?

Reply:
No. for a list put request, you will give a list of value, so it should be `values`.

For a string put request, it's a single value, right?

Reply:
Fine.